frontend/scripts/react-components/shared/url-serializer/url-serializer.component.jsx

```diff
@@ -1,19 +1,43 @@
 import { useEffect } from 'react';
 import pickBy from 'lodash/pickBy';
-import { DONT_SERIALIZE } from 'constants';
+import mapValues from 'lodash/mapValues';
+import isEmpty from 'lodash/isEmpty';
+
+const defaultStringify = (value, DONT_SERIALIZE) => {
+  if ((typeof value !== 'boolean' && isEmpty(value)) || !value) {
```

Comment:
This will change in the next PR, the defaultStringify wont serialize empty objects and empty arrays. And in the new version I changed this cause `isEmpty(number)` was true 🤦‍♂ so the change is to be explicit about objects and arrays.